src/components/ButtonTheme.vue

```diff
@@ -3,12 +3,12 @@ const { toggleUserTheme, getThemeIcon } = useSkin();
 </script>
 
 <template>
-  <ButtonSidebar
+  <RoundButton
```

Comment:
To maintain consistency and improve readability, we should adhere to the [Vue style guide](https://vuejs.org/style-guide/rules-strongly-recommended.html) for naming components. The guide recommends a specific naming convention that helps developers understand the hierarchy and relationships between components more easily. For example, instead of using the component name <RoundButton />, we should rename it to <ButtonRound />. If the button is a base component, it should be named <BaseButtonRound>. This approach allows us to group related components together, designate base components clearly, and quickly identify their purpose within the application.